scylladb/sphinx-scylladb-theme
# Improve bullet points within admonitions style

Closes #904 

## How to test

1. Build the docs
2. Open the page Examples > Admonitions.
3. Lists within admonitions should render as follows:

![image](https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/94d557cc-cd58-4a90-a328-11b0e45d842e)
